packages/SwingSet/src/liveslots/watchedPromises.js

```diff
@@ -116,10 +121,10 @@ export function makeWatchedPromiseManager(
           const [watcher, ...args] = watch;
           void Promise.resolve().then(() => {
             if (watcher.onRejected) {
-              watcher.onRejected(VatUpgradedMessage, ...args);
+              watcher.onRejected(disconnectObjectCapData, ...args);
             } else {
               // eslint-disable-next-line no-throw-literal
-              throw VatUpgradedMessage;
+              throw disconnectObjectCapData;
```

Comment:
I believe we can remove the `no-throw-literal` eslint-husher now

Reply:
Also, throwing capdata is a little weird, I think maybe we should `throw m.unserialize(disconnectObjectCapData)` instead.

It doesn't matter much, we're only doing this to feed an unhandled-rejection handler, and the diagnostic output it emits is just as likely to be useful with dumping capdata as it is dumping a disconnection object.